designs/sam_validate.md

```diff
@@ -0,0 +1,58 @@
+# `sam validate` command
+
+This is the design to improve the `sam validate` command to utilise the the `cfn-python-lint` package.
```

Comment:
Since we don't yet have a design doc for validate, what if we just say this is the design for the command? We started a pattern with creating these with `sam build` and did the same for `sam publish`.

Reply:
That was the point of this PR, implementation will be in another PR :)

Reply:
I know implementation will come later. I was more referring to making this the design for the command. We don't yet have one.

So it this wouldn't be a design to improve but would just be the design for.